app/serializers/api/v3/ecf/participant_serializer.rb

```diff
@@ -141,6 +141,16 @@ def latest_induction_record(user:, profile:, lead_provider:)
             }
           }.compact
         end
+
+        attribute(:participant_id_changes, if: -> { FeatureFlag.active?(:participant_id_changes) } ) do |object|
+          (object.participant_id_changes || []).map do |c|
```

Comment:
could we order by `created_at`?

Reply:
I've added sort on the association:
https://github.com/DFE-Digital/early-careers-framework/pull/4138/files#diff-9802ca3c9c4cf89904fd44bc114e35ebdf2c5dd3d5b645491e2b253e1afef29bR42